sharding/nodesCoordinator/numberOfShardsComputer_test.go

```diff
@@ -3,6 +3,7 @@ package nodesCoordinator
 import (
 	"testing"
 
+	"github.com/multiversx/mx-chain-core-go/core"
```

Comment:
Name of this test in file `TestSovereignNumberOfShardsComputer_ComputeNumberOfShards` is using `sovereign` keyword, whereas we actually test the one from normal chain run type

Reply:
fixed